tests/data/make_central_repo.sh

```diff
@@ -4,24 +4,24 @@
 # match.
 
 # This export file was created with this command, and then added a comment on the first line:
-# make_preloaded_export.py --src-rep $AP_VERIFY_CI_HITS2015_DIR
+# make_preloaded_export.py --src-rep $AP_VERIFY_CI_HITS2015_DIR/preloaded
 butler create central_repo
 butler import --export-file export.yaml --transfer copy central_repo/ $AP_VERIFY_CI_HITS2015_DIR/preloaded/
 
-butler collection-chain central_repo refcats refcats/gen2
-butler collection-chain central_repo DECam/templates templates/deep
+butler collection-chain central_repo refcats refcats/DM-28636
+butler collection-chain central_repo DECam/templates u/elhoward/DM-38243/templates/20230320T214145Z
 butler collection-chain central_repo DECam/defaults DECam/calib,refcats,DECam/templates,skymaps
 
 # Empty out files and make them size 0 in the registry.
 # We do not empty the camera description in ``DECAM/calib/unbounded`
 # because we need it to load the camera geometry.
 find central_repo/refcats -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
-find central_repo/templates -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
+find central_repo/*/*/*/templates -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
```

Comment:
I don't know if this will work with `find`, but if so it should be more robust:
```suggestion
find central_repo/**/templates -name "*.fits" -execdir sh -c '> "$1"' -- {} \;
```

Reply:
That doesn't work, but this works 
```find central_repo -name "*templates*fits" -execdir sh -c '> "$1"' -- {} \;```
and will use this.